### PR TITLE
Fix powershell error on events 40961 and 40962

### PR DESCRIPTION
**Pull request by bjmcnic**
## Proposed commit message
Fix pipeline_error for powershell_operational events 40961 and 40962.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## Related issues

-

### CODE REVIEW

File: packages/windows/data_stream/powershell_operational/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -537,6 +537,7 @@ processors:
       description: Remove all empty values from event_data.
       lang: painless
       source: ctx?.winlog?.event_data?.entrySet().removeIf(entry -> entry.getValue() == null || entry.getValue().equals(""));
+      if: ctx?.winlog?.event_data != null
```
Comment (leehinman) [inline packages/windows/data_stream/powershell_operational/elasticsearch/ingest_pipeline/default.yml:540]: For sure we need to add this to:

forwarded/elasticsearch/ingest_pipeline/powershell_operational.yml

but it probably applies to:
- `powershell/elasticsearch/ingest_pipeline/default.yml`
- `sysmon_operational/elasticsearch/ingest_pipeline/default.yml`
- `forwarded/elasticsearch/ingest_pipeline/powershell.yml`
- `forwarded/elasticsearch/ingest_pipeline/sysmon_operational.yml`

Do you happen to have any sample events you could add to `powershell_operational/_dev/test/pipeline/test-events.json` ?  That way the tests will catch this if we break it in the future.

**Review by leehinman — CHANGES_REQUESTED** [review]
missing changes to `manifest.yml` that match the change in `changelog.yml`

**Comment by jamiehynds** [comment]
Hi @bjmcnic - a customer has been in touch looking for an update on this PR. Their main focus is 40961 and 40962 event support within the `powershell_operational` pipeline? Is it possible to prioritise that pipeline first, and handle the forwarded events (and other) pipelines separately?

File: packages/windows/data_stream/powershell_operational/elasticsearch/ingest_pipeline/default.yml (hunk 1 above)
Comment (bjmcnic) [reply packages/windows/data_stream/powershell_operational/elasticsearch/ingest_pipeline/default.yml:540]: @leehinman Thanks for the tips.  I was able to find sample events for `powershell_operational` and added them to tests.  I also made the analogous fix `powershell` and `sysmon_operational` as you'd suggested, but was unable to create real events that would trigger the issue for those.

**Comment by bjmcnic** [comment]
@jamiehynds I added the tests for `powershell_operational` and the forwarded events for that.  I was unable to naturally generate erring events for `powershell` or `sysmon_operational`, but I put the fixes in anyway.  We'll see what reviewers think.

**Review by leehinman — APPROVED** [review]
LGTM. Thanks for adding the test cases.

**Review by marc-gr — APPROVED** [review]

**Comment by maggieghamry** [comment]
Just as a head's up, it appears there may be a bug (confirmed in `2.1.0` but all Powershell event dashboards are now empty) after installation

**Pull request merged**